modules/inject/src/main/scala/io/janstenpickle/trace4cats/inject/Trace.scala

```diff
@@ -43,6 +43,17 @@ trait Trace[F[_]] {
 }
 
 object Trace extends TraceInstancesLowPriority {
+  trait WithContext[F[_]] extends Trace[F] {
```

Comment:
I think `IOLocal` instance also should be enhanced?

Reply:
Oops, forgot that PR got merged